### PR TITLE
yacc: add page

### PR DESCRIPTION
**Pull request by jahigd02**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution and welcome to tldr. I would suggest reading the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) before contributing in future. 

~~I will update this page later today to comply with the style guide.~~ Thanks for the edit, I will review it soon.

**Comment by jahigd02** [comment]
Thank you @kbdharun! I appreciate the assistance. I ran the linter so it should be O.K. now.

**Comment by kbdharun** [comment]
@jahigd02 Made some modifications to the page, can you check if it looks good to you?

**Review by jahigd02 — COMMENTED** [review]
Looks great :) thanks for all the help

File: pages/common/yacc.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+`yacc -d {{path/to/grammar_file.y}}`
+
+- Compile a grammar file containing the description of the parser and a report of conflicts generated by ambiguities in the grammar:
+
+`yacc -d {{path/to/grammar_file.y}} -v`
+
+- Compile a grammar file, replace the 'y' prepending the output files with '{{prefix}}':
+
+`yacc -d {{path/to/grammar_file.y}}.y -v -b {{prefix}}`
```
Comment (kbdharun) [inline pages/common/yacc.md:17]:
```suggestion
`yacc -d {{path/to/grammar_file.y}} -v -b {{prefix}}`
```

File: pages/common/yacc.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Create a file `y.tab.c` containing the C parser code and compile the grammar file with all necessary constant declarations for values. (Constant declarations file `y.tab.h` is created only when the `-d` flag is used):
+
+`yacc -d {{path/to/grammar_file.y}}`
+
+- Compile a grammar file containing the description of the parser and a report of conflicts generated by ambiguities in the grammar:
+
+`yacc -d {{path/to/grammar_file.y}} -v`
+
+- Compile a grammar file, replace the 'y' prepending the output files with '{{prefix}}':
```
Comment (acuteenvy) [inline pages/common/yacc.md:15]:
```suggestion
- Compile a grammar file, and prefix output filenames with `prefix` instead of `y`:
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**